contrib/devtools/lint-python.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Can you add a copyright notice and comment please? Just copy-paste from lint-whitespace.sh and change the comment to be something like:

```
#
# Copyright (c) 2017 The Bitcoin Core developers
# Distributed under the MIT software license, see the accompanying
# file COPYING or http://www.opensource.org/licenses/mit-license.php.
#
# Check for specified flake8 warnings in python files.
```